### DISCUSSION TITLE
Casual Inference

### DISCUSSION THREAD

**Discussion by btanasie** [McGill-MMA-EnterpriseAnalytics/Dazed-Confusion-Matrix#13] [Ideas]
Looking to use causal inference to improve the understanding of our model and data.

Some potential ideas:  
* Controle for features and their impact of categorizing descriptions
* Look at demographic data and influence factors to explore data bias

[**DoWhy**](https://microsoft.github.io/dowhy/ ) would the main tool for this

**Comment by Andrea-Yzeiri** [comment]
As mentioned in class, because of the nature of our predictive model, predicting on description (multi-class categorical), Uplift tree would be better.

**Reply by btanasie** [reply]
Yes agreed! Ended going with a new package called CausalLift (https://github.com/Minyus/causallift) to perform this for each demographic variable as treatment.
